package.json

```diff
@@ -16,7 +16,8 @@
     "numeric"
   ],
   "scripts": {
-    "test": "istanbul cover  jasmine-node spec"
+    "test": "istanbul cover jasmine-node spec",
+    "wntest": "istanbul cover node_modules/jasmine-node-xt/bin/jasmine-node spec"
```

Comment:
Btw, @codingstill , what is the purpose of this script?

Reply:
@codingstill I mean, I feel this is redundant, given that we defined `jasmine-node-xt` in our `devDependencies` here. Am I missing something?

Reply:
Hi Teo!
`npm run test` gives an error in Windows, so I added `npm run wntest` to be able to run the tests.

Reply:
Cool, good to know, thanks!